pinot-core/src/test/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManagerTest.java

```diff
@@ -765,6 +768,20 @@ public void testOnlyOneSegmentHoldingTheSemaphoreForParticularPartition()
     Assert.assertEquals(secondSegmentDataManager.get().getPartitionGroupConsumerSemaphore().availablePermits(), 1);
   }
 
+  @Test
+  public void testDifferentSegmentBuildTimeLeaseExtenderDestroyedForTables()
```

Comment:
What's the meaning of this test name? What/How to destroy? Can you add some comments to the test?

Reply:
Updated the test with better name.